code/modules/food/food/snacks_vr.dm

```diff
@@ -750,3 +750,14 @@
 	)
 	can_hold = list(/obj/item/weapon/reagent_containers/food/snacks/cube/protein,
 					/obj/item/weapon/reagent_containers/food/snacks/cube/nutriment)
+
+/obj/item/weapon/reagent_containers/food/snacks/carpmeat/sif //Making fish meat non-toxic!  As advised by Ascian!
+    toxin_type = null
+    toxin_amount = null
+
+/obj/item/weapon/reagent_containers/food/snacks/carpmeat/sif/murkfish
+    toxin_type = null
+
+/obj/item/weapon/reagent_containers/food/snacks/carpmeat/fish
+	toxin_type = null
+    toxin_amount = null
```

Comment:
Inconsistant indent on this line

Reply:
That indent appeared somewhere between my computer and Github, let me check...